build/templates/template-bot-resources-legacy-python-pre.json

```diff
@@ -0,0 +1,246 @@
+         "type":"Microsoft.Web/sites",
+         "apiVersion":"2016-08-01",
+         "name":"[variables('webAppName')]",
+         "location":"[variables('resourcesLocation')]",
+         "dependsOn":[
+            "[resourceId('Microsoft.Web/serverfarms', variables('servicePlanName'))]"
+         ],
+         "kind":"app,linux",
+         "properties":{
+            "enabled":true,
+            "hostNameSslStates":[
+               {
+                  "name":"[concat(parameters('newWebAppName'), '.azurewebsites.net')]",
+                  "sslState":"Disabled",
+                  "hostType":"Standard"
+               },
+               {
+                  "name":"[concat(parameters('newWebAppName'), '.scm.azurewebsites.net')]",
+                  "sslState":"Disabled",
+                  "hostType":"Repository"
+               }
+            ],
+            "serverFarmId":"[resourceId('Microsoft.Web/serverfarms', variables('servicePlanName'))]",
+            "reserved":true,
+            "scmSiteAlsoStopped":false,
+            "clientAffinityEnabled":false,
+            "clientCertEnabled":false,
+            "hostNamesDisabled":false,
+            "containerSize":0,
+            "dailyMemoryTimeQuota":0,
+            "httpsOnly":false,
+            "siteConfig":{
+               "appSettings":[
+                  {
+                     "name":"MicrosoftAppId",
+                     "value":"[parameters('appId')]"
+                  },
+                  {
+                     "name":"MicrosoftAppPassword",
+                     "value":"[parameters('appSecret')]"
+                  },
+                  {
+                     "name":"SCM_DO_BUILD_DURING_DEPLOYMENT",
+                     "value":"true"
+                  }
+               ],
+               "cors":{
+                  "allowedOrigins":[
+                     "https://botservice.hosting.portal.azure.net",
+                     "https://hosting.onecloud.azure-test.net/"
+                  ]
+               }
+            }
+         }
+      },
+      {
+         "type":"Microsoft.Web/sites/config",
+         "apiVersion":"2016-08-01",
+         "name":"[concat(variables('webAppName'), '/web')]",
+         "location":"[variables('resourcesLocation')]",
+         "dependsOn":[
+            "[resourceId('Microsoft.Web/sites', variables('webAppName'))]"
+         ],
+         "properties":{
+            "numberOfWorkers":1,
+            "defaultDocuments":[
+               "Default.htm",
+               "Default.html",
+               "Default.asp",
+               "index.htm",
+               "index.html",
+               "iisstart.htm",
+               "default.aspx",
+               "index.php",
+               "hostingstart.html"
+            ],
+            "netFrameworkVersion":"v4.0",
+            "phpVersion":"",
+            "pythonVersion":"",
+            "nodeVersion":"",
+            "linuxFxVersion":"PYTHON|3.7",
```

Comment:
Hi, just one small change for python, @axelsrz suggested we use 3.8 going forward, can we do that?

(please update this on all the other places we we use 3.7)

Thanks